awswrangler/cloudwatch.py

```diff
@@ -79,15 +79,19 @@ def start_query(
     _logger.debug("start_timestamp: %s", start_timestamp)
     _logger.debug("end_timestamp: %s", end_timestamp)
 
-    _validate_args(start_timestamp=start_timestamp, end_timestamp=end_timestamp)
+    _validate_args(
+        start_timestamp=start_timestamp,
+        end_timestamp=end_timestamp,
+    )
     args: Dict[str, Any] = {
-        "logGroupNames": log_group_names,
+        "logGroupIdentifiers": log_group_names,
```

Comment:
So `logGroupNames` and `logGroupIdentifiers` resolve to the same thing? Why does the boto3 API [surface](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/logs/client/start_query.html) both then?

Reply:
Yap, `logGroupIdentifiers` supports both:
> You can specify them by the log group name or ARN.

I'm not sure why. My guess is that they wanted `logGroupNames` as a way to prevent accidental cross-account access. In addition to `logGroupNames`, they also have `logGroupName`, which accepts one string rather than a list of strings. All three parameters can just be reduced to a single one.

I felt like we had a good opportunity to make the API simpler here, rather than our API methods also having `logGroupNames` and `logGroupIdentifiers`. However, I'm happy to change it so that we also have the two parameters.

Reply:
Ok, to be honest I see confusion with both approaches haha If we go with a single argument, it's a simpler API but we have to explain somehow to the user that it accepts both (via the function documentation).

Happy with either though. @kukushking wdyt?

Reply:
Same, not too opinionated about it but current implementation looks good to me.